<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/window_icon.png</normaloff>resources/window_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(25, 26, 43);
}
QMenuBar{
	background-color: rgb(234, 237, 244);
	color: rgb(25, 26, 43);
}
QMenuBar::item:selected{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu::item:enabled:selected{
	background-color: rgb(234, 237, 244);
	color: rgb(25, 26, 43);
}
QMenu#menuEdge_Enhance, #menuFind_Edge, #menuSmooth, #menuAbout{
	background-color: rgb(234, 237, 244);
	color: rgb(25, 26, 43);
}
QMenu::item:enabled:selected#menuEdge_Enhance{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu::item:enabled:selected#menuFind_Edge{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu::item:enabled:selected#menuSmooth{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu::item:enabled:selected#menuAbout{
	background-color: rgb(231, 34, 88);
	color: rgb(234, 237, 244);
}
QMenu::item:disabled{
	color: gray;
}
QStatusBar{
	color: white;
}
QPushButton{
	color: rgb(25, 26, 43);
	 border-style: none;
}
QPushButton:hover{
	border: 1px solid #191a2b;
	border-radius: 10px;
}
QPushButton:pressed{
	border: 2px solid #e72258;
	border-radius: 10px;
}
QPushButton:hover#btnOpen{
	border-style: none;
}
QPushButton:pressed#Open{
	border-style: none;
}
QSlider::handle:horizontaly{
	background-color:  rgb(25, 26, 43);
	border: 1px solid #191a2b;
	width: 9px;
	margin: -6px 0;
	border-radius: 5px;
}
QSlider::groove:horizontaly#gammaSlider{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(27, 27, 27, 255), stop:1 rgba(207, 207, 207, 255));
	height: 10px;
	border-radius: 5px;
}
QSlider::groove:horizontaly#transpaSlider{
	height: 10px;
	border-radius: 5px;
}
QSlider::sub-page:horizontaly#transpaSlider{
	background: rgb(231, 34, 88);
	border-radius: 5px;
}
QSlider::add-page:horizontaly#transpaSlider{
	background: rgba(214, 214, 214);
	border-radius: 5px;
}
QSpinBox{
	color: rgb(25, 26, 43);
}
QDoubleSpinBox{
	color: rgb(25, 26, 43);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>211</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 237, 244);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0,1,0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e72258;&quot;&gt;Rotate &amp;amp; Adjustments&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="horizontalWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnRotate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Rotate 90Â° Left</string>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resources/rotate.png</normaloff>resources/rotate.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFlipH">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Flip Horizontaly</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resources/flip_h.png</normaloff>resources/flip_h.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFlipV">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Flip Verticaly</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resources/flip_v.png</normaloff>resources/flip_v.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#191a2b;&quot;&gt;Gamma Correction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="gammaSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>22</height>
          </size>
         </property>
         <property name="minimum">
          <number>25</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="gammaSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#191a2b;&quot;&gt;Transparency (%)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="transpaSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="transpaSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Revert You Image to the Original</string>
       </property>
       <property name="text">
        <string>  Reset</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resources/reset.png</normaloff>resources/reset.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>311</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Open an Image...</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/open_image_icon.png</normaloff>resources/open_image_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>331</width>
      <height>311</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>761</width>
      <height>421</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>imageLabel</zorder>
   <zorder>verticalWidget</zorder>
   <zorder>btnOpen</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Fil&amp;ters</string>
    </property>
    <widget class="QMenu" name="menuEdge_Enhance">
     <property name="title">
      <string>Edge Enhance</string>
     </property>
     <addaction name="actionE_E_Normal"/>
     <addaction name="actionE_E_More"/>
    </widget>
    <widget class="QMenu" name="menuSmooth">
     <property name="title">
      <string>Smooth</string>
     </property>
     <addaction name="actionS_Normal"/>
     <addaction name="actionS_More"/>
    </widget>
    <widget class="QMenu" name="menuFind_Edge">
     <property name="title">
      <string>Find Edge</string>
     </property>
     <addaction name="actionF_E_Weak_Detection"/>
     <addaction name="actionF_E_Medium_Detection"/>
     <addaction name="actionF_E_Strong_Detection"/>
    </widget>
    <addaction name="actionBlack_and_White"/>
    <addaction name="actionGray_Scale"/>
    <addaction name="actionNegative"/>
    <addaction name="separator"/>
    <addaction name="actionBlur"/>
    <addaction name="actionCountor"/>
    <addaction name="actionDetail"/>
    <addaction name="menuEdge_Enhance"/>
    <addaction name="actionEmboss"/>
    <addaction name="menuFind_Edge"/>
    <addaction name="actionSharpen"/>
    <addaction name="menuSmooth"/>
    <addaction name="separator"/>
    <addaction name="actionRed_Scale"/>
    <addaction name="actionGreen_Scale"/>
    <addaction name="actionBlue_Scale"/>
    <addaction name="separator"/>
    <addaction name="actionLogarithmic"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>&amp;About</string>
     </property>
     <addaction name="actionAbout_App"/>
     <addaction name="actionAbout_Img"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionHide_Text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hide Text With SHA256</string>
   </property>
  </action>
  <action name="actionIdentify_Secret_Text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Identify Secret Text</string>
   </property>
  </action>
  <action name="actionBlack_and_White">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Black and White</string>
   </property>
  </action>
  <action name="actionGray_Scale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gray Scale</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionCountor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Countor</string>
   </property>
  </action>
  <action name="actionDetail">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detail</string>
   </property>
  </action>
  <action name="actionE_E_Normal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionE_E_More">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </action>
  <action name="actionEmboss">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Emboss</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
  <action name="actionS_Normal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionS_More">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </action>
  <action name="actionF_E_Weak_Detection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Weak Detection</string>
   </property>
  </action>
  <action name="actionF_E_Medium_Detection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Medium Detection</string>
   </property>
  </action>
  <action name="actionF_E_Strong_Detection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Strong Detection</string>
   </property>
  </action>
  <action name="actionAbout_Image">
   <property name="text">
    <string>About Image</string>
   </property>
  </action>
  <action name="actionAbout_App">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>App</string>
   </property>
  </action>
  <action name="actionAbout_Img">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRed_Scale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Red Scale</string>
   </property>
  </action>
  <action name="actionGreen_Scale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Green Scale</string>
   </property>
  </action>
  <action name="actionBlue_Scale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Blue Scale</string>
   </property>
  </action>
  <action name="actionLogarithmic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logarithmic</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
